### PR TITLE
Add dims as k8s maintainer

### PR DESCRIPTION
**Pull request by dims**
I was wondering why @derekwaynecarr was able to edit my PR to update a release note and i could not edit other's PR and found this group.

Also, we may need to prune this group at some point (cc @kubernetes/owners)

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Review by justaugustus — CHANGES_REQUESTED** [review]
@dims -- I'm actually surprised you weren't in one of the write access groups already!

In two separate commits, can you:
- Migrate this team to the SIG Release config subdirectory
- Add yourself in

Overall, we're trying to minimize the folks in `kubernetes-maintainers`, but that conversation with @spiffxp trailed off a while ago (ref: https://github.com/kubernetes/enhancements/issues/590).

That said, I do think you should be one of the folks in this group.
/hold for fixes

**Comment by cblecker** [comment]
I have no issue adding dims to this group, albeit, it's legacy and we don't have a process for becoming a "maintainer" anymore.



> * Migrate this team to the SIG Release config subdirectory

@justaugustus What's your thought process behind giving sig-release control of this legacy group?

**Comment by dims** [comment]
/assign @cblecker @spiffxp @mrbobbytables  

(adding more people for visibility)

**Comment by dims** [comment]
@justaugustus i don't want to do that :) i am happy to wait until the discussion settles and would like the approvers to chime in.

**Comment by justaugustus** [comment]
> @justaugustus What's your thought process behind giving sig-release control of this legacy group?

@cblecker -- Usually we'd put a `<repo>-maintainers` team within the SIG directory that owns it.
SIG Release cares about the content of the repo (and who has write access), but SIG Architecture approves repos in the `kubernetes` org, so really you could make a case for it living in either subdirectory.

This is more so for continued maintenance/eventual pruning of the group, without needing a top-level `/approve`.
(`milestone-maintainers` would be the parallel group that SIG Release maintains.)

**Comment by cblecker** [comment]
> This is more so for continued maintenance/eventual pruning of the group, without needing a top-level `/approve`.

As this is a legacy group that no one sig technically owns, IMO I'm inclined to keep it with the @kubernetes/owners group.

**Comment by dims** [comment]
i'd agree with you @cblecker

**Comment by justaugustus** [comment]
Cool beans! :two_hearts: 
/lgtm

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

**Review by palnabarun — APPROVED** [review]
/lgtm

**Comment by dims** [comment]
/hold cancel

thanks everyone!

**Pull request merged**
